pyxnat/tests/resources_test.py

```diff
@@ -310,6 +311,8 @@ def test_get_zip():
     r.get(local_dir, extract=True)
     for f in file_list:
         assert os.path.exists(f)
+    r.get(local_dir, extract=False)
```

Comment:
Assertion for downloaded resource missing?

Reply:
Fixed in b5ef950.